scripts/cli-create-table-storage.sh

```diff
@@ -0,0 +1,30 @@
+LOCATION='eastus'
+RESOURCE_GROUP_NAME='rg-msdocs-tables-sdk-demo'
+STORAGE_ACCOUNT_NAME='stmsdocstablessdkdemo'
```

Comment:
Does this also need to be unique in Azure?

Reply:
If we are providing a ps1 script for creating a COSMOS resource, I think we should also provide one for creating a storage resource.